README.md

```diff
@@ -25,7 +29,10 @@ The Cloud Controller supports Postgres and Mysql.
 
 The Cloud Controller manages a blobstore for:
 
+All Platforms:
 * Resource cache: During package upload resource matching, Cloud Controller will only upload files it doesn't already have in this cache.
+
+CF for VMs only (i.e. not on kubernetes):
```

Comment:
I think it's more accurate to say "capi-release only." Kubecf & friends do use a blobstore for everything.